lib/arch/generic/codec.c

```diff
@@ -19,10 +19,6 @@ BASE64_ENC_FUNCTION(plain)
 BASE64_DEC_FUNCTION(plain)
 {
 	#include "dec_head.c"
-#if BASE64_WORDSIZE == 32
-	#include "32/dec_loop.c"
-#elif BASE64_WORDSIZE == 64
-	#include "64/dec_loop.c"
-#endif
+	#include "dec_loop.c"
 	#include "dec_tail.c"
```

Comment:
Hmm. There's an underlying assumption here that the wordsize is always going to be >= 32 bit. The current code has no such assumption and will work on an 8-bit micro (untested, but in principle...)

Why not retain the `#if BASE64_WORDSIZE >= 32`?